en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics.md

```diff
@@ -78,6 +78,119 @@ Here are some of the graphs generated in Choreo cloud.
 
 [![Choreo Analytics Latency]({{base_path}}/assets/img/deploy/choreo-analytics-latency.png)]({{base_path}}/assets/img/deploy/choreo-analytics-latency.png)
 
+## ELK Analytics for Choreo Connect
```

Comment:
Shall we introduce the two main ways at the top of the page? Right after the intro paragraph? We can link the two subsections too. Then the user wouldn't have to scroll to find it.